### ISSUE TITLE
The Methods Used

### ISSUE THREAD

**Issue by RoyiAvital** [ducha-aiki/ransac-tutorial-2020-data#1]
I read your great tutorial [Benchmarking Robust Estimation Methods](https://cmp.felk.cvut.cz/cvpr2020-ransac-tutorial/presentations/RANSAC-CVPR20-Mishkin.pdf).

Looking at Homography, I couldn't decipher what were the exact methods used.

![image](https://github.com/ducha-aiki/ransac-tutorial-2020-data/assets/7347975/3b428c66-9f33-404f-94d1-d766663d5f72)

1. SciKit Image  
   Do you mean something like: `model_robust, inliers = ransac((src, dst), ProjectiveTransform, min_samples=3, residual_threshold=2, max_trials=100)`?
2. LMeDS and OpenCV Rho means using methods `LMEDS` and `RHO - PROSAC` for OpenCV?
3. What's `OpenCV H`? Is it just the Least Squares method with added robustness?
4. What's `GC` on its own? Is that `RANSAC GC`? Namely Graph Cut RANSAC?

**Comment by ducha-aiki** [comment]
Hi,


> I read your great tutorial [Benchmarking Robust Estimation Methods](https://cmp.felk.cvut.cz/cvpr2020-ransac-tutorial/presentations/RANSAC-CVPR20-Mishkin.pdf).
> 
> Looking at Homography, I couldn't decipher what were the exact methods used.
> 
> ![image](https://user-images.githubusercontent.com/7347975/278829011-3b428c66-9f33-404f-94d1-d766663d5f72.png)
> 
> 1. SciKit Image
>    Do you mean something like: `model_robust, inliers = ransac((src, dst), ProjectiveTransform, min_samples=3, residual_threshold=2, max_trials=100)`?
Yes, but your params are wrong. Minsamples 4, num trials as in slide, threshold tuned to be optimal
> 2. LMeDS and OpenCV Rho means using methods `LMEDS` and `RHO - PROSAC` for OpenCV?
Yes
> 3. What's `OpenCV H`? Is it just the Least Squares method with added robustness?
No. It is cv2.RANSAC flag
> 4. What's `GC` on its own? Is that `RANSAC GC`? Namely Graph Cut RANSAC?
Yes

**Comment by RoyiAvital** [comment]
I see.

It makes one wonder why would using RANSAC with SciKit Image will have different results from OpenCV.
As the only difference, for vanilla RANSAC, should be the random samples index generation.

Your work is greatly appreciated as a review of the different methods.

**Comment by ducha-aiki** [comment]
> I see.
> 
> It makes one wonder why would using RANSAC with SciKit Image will have different results from OpenCV. As the only difference, for vanilla RANSAC, should be the random samples index generation.

No. OpenCV also implements quick sample rejection based on the cross-check
https://github.com/opencv/opencv/blob/617d7ff575200c0a647cc615b86003f10b64587b/modules/calib3d/src/fundam.cpp#L67
> 
> Your work is greatly appreciated as a review of the different methods.

Thank you
